frontend/src/App.js

```diff
@@ -12,25 +12,32 @@ class App extends Component {
   state = {
     sources: [],
     err: null,
+    version: '',
   }
 
   componentDidMount() {
     this.loadSources()
+    this.localDbVersion()
```

Comment:
Still the same, it doesn't describe an action, `loadDbVersion()` would be more appropriate.

Reply:
confused?

Reply:
No, since it doesn't describe an action, it simply reads `local db version`

Reply:
How does `getLocalDbVersion()` sound?

Reply:
just `getDbVersion()` is ok too, or `loadDbVersion` (inline with `loadSources()`). I prefer the latter for the aforementioned reason.